workspaces/src/operations.rs

```diff
@@ -231,14 +234,14 @@ impl<'a, 'b, T: Network> CallTransaction<'a, 'b, T> {
     /// Similiar to `args`, specify an argument that is JSON serializable and can be
     /// accepted by the equivalent contract. Recommend to use something like
     /// `serde_json::json!` macro to easily serialize the arguments.
-    pub fn args_json<U: serde::Serialize>(mut self, args: U) -> anyhow::Result<Self> {
+    pub fn args_json<U: serde::Serialize>(mut self, args: U) -> Result<Self, BytesError> {
         self.function = self.function.args_json(args)?;
         Ok(self)
     }
```

Comment:
Consider making this method infalible and using `reqwest` approach of postponing the error to the point where the request is made: 

https://docs.rs/reqwest/latest/src/reqwest/async_impl/request.rs.html#442-455